### PR TITLE
fix integration test section of the docs

### PR DESCRIPTION
**Pull request by mubarak23**
- PR For issue: #789 
- Change the folder structure of the code that show how integration test is done by 
moving test folder into the root of the project, where both SRC and TESTS are in the same tree,
- Make the changes to reflect in the md file
- Runs test to make to sure everything works

- - -
This change is [Reviewable](https://reviewable.io/reviews/cairo-book/cairo-book/802)

### CODE REVIEW

File: src/ch10-02-test-organization.md [hunk 1]
```diff
@@ -31,13 +31,13 @@ Integration tests use your library in the same way any other code would. Their p
...
-├── Scarb.toml
+├── Cargo.lock
+├── Cargo.toml
 ├── src
-│   ├── lib.cairo
-│   ├── tests
-│   │   └── integration_test.cairo
-│   └── tests.cairo
+│   └── lib.cairo
+│   └── tests.cairo
```
Comment (enitrat) [inline src/ch10-02-test-organization.md:38]: no longer a `test.cairo` file here

File: src/ch10-02-test-organization.md [hunk 2]
```diff
@@ -31,13 +31,13 @@ Integration tests use your library in the same way any other code would. Their p
 ### The _tests_ Directory
 
 ```shell
-adder
-├── Scarb.toml
+├── Cargo.lock
```
Comment (enitrat) [inline src/ch10-02-test-organization.md:34]: Cargo? 
restore the previous structure with the correct package name (using the `tree -L 2` command at the root)

File: listings/ch10-testing-cairo-programs/no_listing_09_integration_test/src/lib.cairo [hunk 3]
```diff
@@ -1,4 +1,4 @@
-fn it_adds_two(a: u8, b: u8) -> u8 {
+pub fn it_adds_two(a: u8, b: u8) -> u8 {
```
Comment (enitrat) [inline listings/ch10-testing-cairo-programs/no_listing_09_integration_test/src/lib.cairo:1]: why pub?

**Review by enitrat — CHANGES_REQUESTED** [review]
Still a lot of content missing - no changes in the markdown files?

**Comment by TAdev0** [comment]
@mubarak23  are you still working on this?

**Comment by enitrat** [comment]
Closing as no activity

**Pull request closed without merging**